Items/04_00_WeaponAttachments.txt

```diff
@@ -474,45 +508,58 @@ This allows three point slings to be used without getting caught on your gear.
 Action. Allows you to carry the gun on your back. Holstering your weapon is a 
 Free Action.
 
-Strength Requirement:
-*  -1
-
-Range Bracket Miss Chance:
-*  -0.5
+Strength Requirement:  -1
+Range Bracket Miss Chance:  -0.5
 
 ===== Level 10 Requirement =====
 
 == Pistol Smart-Lanyard Holster System  $250 ==
+
     Dropping your weapon holsters it as free action. Drawing it takes 1 Action 
 as normal.
 
 ==============================
 Ammunition / Magazine Accessories
 ==============================
-    Only extended, magpull, and Drum mags apply to unconventional weapons 
-(weapons other than bullet-firing.) Total mag price should be calculated by 
-taking multipliers for extended mags first, then multipliers for special effect, 
-then magazine accessories like magpulls. Multipliers are multiplied together but 
-they do not multiply the constant.
-
-    For example, extended mag + poison bullet magazine magazine cost is 
-(1.3x * 2x + 50) = (2.6x+50)
 
 ===== Level 1 Requirement =====
 
 == Extended Magazines  $(1.3x) Mag cost. ==
+
      $12 mags now cost $15, etc. Round Up. Adds 30% magazine capacity to 
 magazine/clip-based weapons. Revolver systems or rocket launchers, for example, 
 are excluded.
 
 == Hollow Point Magazine  $1.5x + 10 mag cost ==
-    +30% base damage, -1 Armor Piercing Level
+
+    Hollow point ammunition are bullets that have either a hollow tip or a tip
+with water inside. This makes the bullet fragment more dramatically on impact,
+which means that it penetrates armor less effectively. However, if it hits 
+something squishy like fruit, a person's body or the delecate silicone and boron
+of a security drone, it shreds it much more effectively.
+
+Base damage: +30%
+Armor Piercing Level: -1
 
 == Armor Piercing Magazine  $1.5x + 10 mag cost ==
-    +1 Armor Piercing Level, -10% base damage
+
+    Armor piercing rounds are made of heavier metal that doesn't fragment as
+easily on impact. This means that it is much more likely to go through armor.
+However, this also means that it is likely to go in one side of a soft target
+and out of the other, which typically damages soft targets like people a lot 
+less.
+
+Armor Piercing Level: +1
+Base Damage: -10%
 
 == ED Electric Discharge Magazine  $2x + 20 magazine cost ==
-    1/2 the bullet's base damage is converted to electrical damage.
+
+    ED rounds contain what basically amounts to a capacitor and electrolytic
+chaff. When the round hits, it shocks whatever it hits. But this chaff does 
+little to fragment the way that conventional bullets do. 
+
+Base Damage: -50%
+Electric Damage: +50%

```

Comment:
So, +50% Electric Damage reads like it only works if your gun already deals Electric damage (if your gun deals zero electric damage base, +50% is still zero). We want to specify that the +50% is derived from the Base Damage.